tutorials/plot_background_statistics.py

```diff
@@ -0,0 +1,496 @@
+                                       tail=1, n_permutations=n_permutations)
+
+###############################################################################
+# We can also combine TFCE and the "hat" correction:
+t_tfce_hat, _, p_tfce_hat, H0 = \
+    spatio_temporal_cluster_1samp_test(X, n_jobs=1, threshold=threshold_tfce,
+                                       connectivity=None, tail=1,
+                                       n_permutations=n_permutations,
+                                       stat_fun=stat_fun_hat, buffer_size=None)
+
+###############################################################################
+# Visualize and compare methods
+# -----------------------------
+# Let's take a look at these statistics. The top row shows each test statistic,
+# and the bottom shows p values for various statistical tests, with the ones
+# with proper control over FWER or FDR with bold titles.
+
+fig = plt.figure(facecolor='w', figsize=(12, 3))
+
+x, y = np.mgrid[0:width, 0:width]
+cmap = 'YlGnBu'
+kwargs = dict(rstride=1, cstride=1, linewidth=0, cmap=cmap)
+
+mccs = [False, False, False,
+        True, True, True,
+        True, True, True, True]
+ts = [t_uncorrected, t_uncorrected_hat, t_uncorrected_perm,
+      None, None, t_perm,
+      t_clust, t_hat, t_tfce, t_tfce_hat]
+titles = ['t statistic', '$\mathrm{t_{hat}}$', 'Permutation',
+          'Bonferroni', 'FDR', '$\mathbf{Perm_{max}}$',
+          'Clustering', r'$\mathbf{C_{hat}}$',
+          r'$\mathbf{C_{TFCE}}$',
+          r'$\mathbf{C_{hat,TFCE}}$']
+for ii, t in enumerate(ts):
+    ax = fig.add_subplot(2, 10, ii + 1, projection='3d')
+    if t is None:
+        fig.delaxes(ax)
+    else:
+        ax.plot_surface(x, y, np.reshape(t, (width, width)), **kwargs)
+        ax.set(xticks=[], yticks=[], zticks=[])
+
+p_lims = [-np.log10(0.05), -np.log10(0.001)]
+ps = [p_uncorrected, p_uncorrected_hat, p_uncorrected_perm,
+      p_bon, p_fdr, p_perm,
+      p_clust, p_hat, p_tfce, p_tfce_hat]
+for ii, (p, title, mcc) in enumerate(zip(ps, titles, mccs)):
+    ax = fig.add_subplot(2, 10, 11 + ii)
+    use_p = -np.log10(np.reshape(np.maximum(p, 1e-5), (width, width)))
+    img = ax.imshow(use_p, cmap=cmap, vmin=p_lims[0], vmax=p_lims[1],
+                    interpolation='bilinear')
+    ax.set(xticks=[], yticks=[], title=title)
+    if mcc:
+        ax.title.set_weight('bold')
+    cbar = plt.colorbar(ax=ax, shrink=0.75, orientation='horizontal',
+                        fraction=0.1, pad=0.025, mappable=img)
+    cbar.set_label('-log10(p)')
+    cbar.set_ticks(p_lims)
+    cbar.set_ticklabels(['%0.1f' % p for p in p_lims])
+
+fig.tight_layout(pad=0, w_pad=0.1, h_pad=0.1)
+plt.show()
+
+###############################################################################
+# The first three columns show the parametric and non-parametric statistics
+# that are not corrected for multiple comparisons:
+#
+# - **t test** has peaky edges.
```

Comment:
t-test (several times in this section)

Reply:
Also, what are "peaky edges"? Is there a better term for what you mean? Jagged maybe?